wgengine/router/router_linux.go

```diff
@@ -129,6 +130,12 @@ func chooseFireWallMode(logf logger.Logf, det tableDetector) linuxfw.FirewallMod
 func newNetfilterRunner(logf logger.Logf) (netfilterRunner, error) {
 	tableDetector := &linuxFWDetector{}
 	mode := chooseFireWallMode(logf, tableDetector)
+	if mode == linuxfw.FirewallModeIPTables {
```

Comment:
Did you mean nftables?

Reply:
I thought nftables user would be more than iptables due to our heuristic preferring nftables. Do you believe otherwise?

Reply:
Right, let's save bytes in the common case